js/label/static/Label.svelte

```diff
@@ -37,18 +34,31 @@
 				}}
 			>
 				<div class="inner-wrap">
-					<div class="bar" style="width: {confidence_set.confidence * 100}%" />
-					<div class="label">
-						<div class="text">{confidence_set.label}</div>
+					<meter
+						aria-labelledby="meter-text"
+						class="bar"
+						min="0"
+						max="100"
+						style="width: {confidence_set.confidence *
+							100}%; background: var(--stat-background-fill);
+						"
+						value={confidence_set.confidence * 100}
+						aria-label={Math.round(confidence_set.confidence * 100) + "%"}
+					/>
+
+					<dl class="label">
```

Comment:
Thought `<dl>` should be the common ancestor of all `<dt>` and `<dd>` pairs as it represents a *list* of descriptions, but not sure it's ok if `<button>`s exist between them like this case 🤔